trie/verkle.go

```diff
@@ -228,7 +226,7 @@ func (trie *VerkleTrie) TryDelete(key []byte) error {
 // Hash returns the root hash of the trie. It does not write to the database and
 // can be used even if the trie doesn't have one.
 func (trie *VerkleTrie) Hash() common.Hash {
-	return trie.root.Commit().Bytes()
+	return common.BytesToHash(trie.root.Commit().Bytes())
```

Comment:
Nothing interesting, just a type conversion from `SerializedPointCompressed` to `common.Hash`, which is "semantic" since both are 32-byte slices.

This is why I wanted to create types for `go-ipa` regarding serialization. There's no way to get a wrong `[]byte` size (compressed vs uncompressed).